### PR TITLE
Update to Task API v0.24.1

### PR DESCRIPTION
**Pull request by mfranciszkiewicz**
Use the `DBTaskManager` from `apputils`

### CODE REVIEW

File: image/golem_blender_app/golem_blender_app/commands/verify.py [hunk 1]
```diff
@@ -1,15 +1,19 @@
 from pathlib import Path
+
 from typing import Tuple, Optional
 import json
 import shutil
 import zipfile
 
+from golem_task_api.apputils.task import SubtaskStatus
```
Comment (maaktweluit) [inline image/golem_blender_app/golem_blender_app/commands/verify.py]: alphabetic ordering would be after `golem_blender_app`

File: image/golem_blender_app/requirements.txt [hunk 2]
```diff
@@ -4,12 +4,12 @@ scikit-learn==0.19.1
...
 Pillow==5.1.0
 sklearn
 numpy==1.15.4
 psutil
 PyWavelets==1.0.2
 opencv-python==4.0.0.21
 
-typing
-golem_task_api==0.24.0
+golem_task_api==0.24.1
```
Comment (maaktweluit) [inline image/golem_blender_app/requirements.txt]: not alphabetically ordered

File: image/golem_blender_app/golem_blender_app/commands/verify.py (hunk 1 above)
Comment (mfranciszkiewicz) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: I'd leave it as is - it's similar to `from . import ...`

Comment (maaktweluit) [reply image/golem_blender_app/golem_blender_app/commands/verify.py]: alphabetic ordering does not know exceptions :P but i'm fine with leaving it as is :)

**Pull request merged**